agent/src/main/java/com/microsoft/hydralab/agent/config/DeviceDriverConfig.java

```diff
@@ -26,15 +25,34 @@
 @Configuration
 public class DeviceDriverConfig {
 
+    @Value("${app.device.monitor.android.enabled:false}")
+    boolean android;
+    @Value("${app.device.monitor.windows.enabled:false}")
+    boolean windows;
+    @Value("${app.device.monitor.ios.enabled:false}")
+    boolean ios;
```

Comment:
Please add enableXXX as prefix for the naming to clarify the boolean variable usage.